specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/textanalytics.json

```diff
@@ -0,0 +1,2254 @@
+        "version": "2021-11-01-preview"
+    },
+    "securityDefinitions": {
+        "apim_key": {
+            "type": "apiKey",
+            "name": "Ocp-Apim-Subscription-Key",
+            "in": "header"
+        }
+    },
+    "security": [
+        {
+            "apim_key": []
+        }
+    ],
+    "x-ms-parameterized-host": {
+        "hostTemplate": "{Endpoint}/language",
+        "useSchemePrefix": false,
+        "parameters": [
+          {
+            "$ref": "#/parameters/Endpoint"
+          }
+        ]
+    },
+    "paths": {
+        "/analyze-text/jobs": {
+            "post": {
+                "consumes": [
+                    "application/json"
+                ],
+                "produces": [
+                    "application/json"
+                ],
+                "description": "Submit a collection of text documents for analysis. Specify one or more unique tasks to be executed.",
+                "operationId": "AnalyzeText_SubmitJob",
+                "summary": "Submit text analysis job",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "description": "Collection of documents to analyze and tasks to execute.",
+                        "in": "body",
+                        "name": "body",
+                        "schema": {
+                            "$ref": "#/definitions/AnalyzeTextJobsInput"
+                        }
+                    }
+                ],
+                "responses": {
+                    "202": {
+                        "description": "A successful call results with an Operation-Location header used to check the status of the analysis job.",
+                        "headers": {
+                            "Operation-Location": {
+                                "type": "string"
+                            }
+                        }
+                    },
+                    "default": {
+                        "description": "Error response.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                },
+                "x-ms-examples": {
+                    "Successful Submit Analysis Job Request": {
+                        "$ref": "./examples/SuccessfulAnalyzeTextJobSubmitRequest.json"
+                    }
+                },
+                "deprecated": false,
```

Comment:
`false` is the default value for `deprecated` so there is no need to specify it explicitly.

Reply:
Thanks for the catch, updated!